react/CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
+## v5.0.1
+
+-   Update theme definitions to use the [@pxblue/colors](https://www.npmjs.com/package/@pxblue/colors) version 3.0.0.
```

Comment:
Would've been great to catch this in the 5.0.0 net 🤒

Reply:
I used to have the same feeling towards all my pull requests until my mega PRs get banned :P